packages/peregrine/lib/talons/CartPage/ProductListing/useProduct.js

```diff
@@ -24,15 +27,32 @@ import { deriveErrorMessage } from '../../../util/deriveErrorMessage';
  * @example <caption>Importing into your project</caption>
  * import { useProduct } from '@magento/peregrine/lib/talons/CartPage/ProductListing/useProduct';
  */
+
 export const useProduct = props => {
+    const { item, setActiveEditItem, setIsCartUpdating } = props;
+
+    const operations = mergeOperations(DEFAULT_OPERATIONS, props.operations);
     const {
-        item,
-        mutations: { removeItemMutation, updateItemQuantityMutation },
-        setActiveEditItem,
-        setIsCartUpdating
-    } = props;
+        removeItemMutation,
+        updateItemQuantityMutation,
+        getConfigurableThumbnailSource
+    } = operations;
 
-    const flatProduct = flattenProduct(item);
+    const { data: getConfigurableThumbnailSourceData } = useQuery(
```

Comment:
I would suggest calling this `configurableThumbnailSourceData` instead.

Reply:
Done,
Thank you